fuel-client/src/client/schema/resource.rs

```diff
@@ -46,18 +46,21 @@ pub struct SpendQueryElementInput {
 pub enum Resource {
     Coin(Coin),
     Message(Message),
+    #[cynic(fallback)]
```

Comment:
If cynic is now requiring a fallback type on enums, do we want to use `Unknown`, or is there another way annotate it as an error value that should never realistically be populated? `Unkown` though also works though, but wondering if there is some way to show that some of these Unknowns should not be used

Reply:
[It](https://cynic-rs.dev/derives/inline-fragments.html#fallbacks) says that it is only one way=(